telemetry/telemetry.go

```diff
@@ -83,23 +83,23 @@ func Init(isLoggingEnabled bool) {
 	var err error
 	analyticsClient, err = analytics.NewClient(
 		analytics.Config{
-			WriteKey:        segmentKey,
-			SegmentEndpoint: segmentEndpoint,
+			AmplitudeAPIKey:   amplitudeKey,
+			AmplitudeEndpoint: amplitudeEndpoint,
 			App: analytics.AppInfo{
 				Name:      "akita-cli",
 				Version:   version.ReleaseVersion().String(),
 				Build:     version.GitVersion(),
 				Namespace: "",
 			},
-			// No output from the Segment library
+			// No output from the Amplitude library
 			IsLoggingEnabled: false,
-			// IsMixpanelEnabled: false,  -- irrelevant for us, leaving at default value
-			BatchSize: 1, // disable batching
```

Comment:
I don't recall precisely why I disabled batching, but I think it had to do with catching errors during command-line execution, or maybe other abnormal exits.  I think that should still be part of our use of Amplitude, if batching can be disabled?

Reply:
We can disable batching we can use `FlushQueueSize` config for this.